<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynthControl</class>
 <widget class="QWidget" name="SynthControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synthesizer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="3">
      <widget class="QPushButton" name="storeButton">
       <property name="text">
        <string>Set as Default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Save to Score</string>
       </property>
       <property name="text">
        <string>Save to Score</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="loadButton">
       <property name="toolTip">
        <string>Load from Score</string>
       </property>
       <property name="text">
        <string>Load from Score</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="recallButton">
       <property name="text">
        <string>Load Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Awl::MeterSlider" name="gain">
       <property name="statusTip">
        <string>Master volume</string>
       </property>
       <property name="channel">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Vol</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="masterEffectsTab">
      <attribute name="title">
       <string>Master Effects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0" colspan="6">
        <widget class="QStackedWidget" name="effectStackA">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Effect B:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="effectA">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QStackedWidget" name="effectStackB"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Effect A:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="effectB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tuning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>Master tuning:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="masterTuning">
         <property name="suffix">
          <string extracomment="Frequency Herz">Hz</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>300.000000000000000</double>
         </property>
         <property name="maximum">
          <double>600.000000000000000</double>
         </property>
         <property name="value">
          <double>440.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>275</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>127</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="changeTuningButton">
         <property name="text">
          <string>Change Tuning</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>getIconPixmap</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Awl::Slider</class>
   <extends>QWidget</extends>
   <header>awl/slider.h</header>
  </customwidget>
  <customwidget>
   <class>Awl::VolSlider</class>
   <extends>Awl::Slider</extends>
   <header>awl/volslider.h</header>
  </customwidget>
  <customwidget>
   <class>Awl::MeterSlider</class>
   <extends>Awl::VolSlider</extends>
   <header>awl/mslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
